kind.yaml

```diff
@@ -0,0 +1,37 @@
+# 
```

Comment:
This is just temporary, my notes on how to spin things up on my local Linux machine.